CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ Extending the adopted spec, each change should have a link to its corresponding
 
 ### Changed
 
+- Make Forseti Real-Time Enforcer VM have private IP by default. [#180]
```

Comment:
This entry must go under the `Unreleased` section which has been readded on the master branch. Please rebase and correct the placement.